ui/src/app/machines/components/HeaderStrip/ActionFormWrapper/OverrideTestForm/OverrideTestForm.test.js

```diff
@@ -54,7 +54,8 @@ describe("OverrideTestForm", () => {
     };
   });
 
-  it("displays message for a single machine with no failed tests", () => {
+  it(`display failed tests warning without suppress tests checkbox for machines
+    with no failed tests`, () => {
```

Comment:
Unless I missed it I think there should also be a test for multiple machines with no failed tests

Reply:
Fixed!